File: 3_Semestre/Estrutura_Dados/Aula01/exercicio02.js
var vetor = new Array(20)

function gerarNum(min, max) {
    min = Math.ceil(min)
    max = Math.floor(max)

    return Math.floor(Math.random() * (max - min) + min)
}

function gerarVetor(vetor, min, max) {
    var vpar = new Array()

    for (var i = 0; i < vetor.length; i++) {
        var temp = gerarNum(min, max)

        if (temp % 2 == 0) {
            vpar.push(temp)
        }

        vetor[i] = temp
    }

    console.log(vetor)
    console.log(vpar)
}

gerarVetor(vetor, 0, 100)

File: 3_Semestre/Estrutura_Dados/Aula01/exercicio02.java
package Aula01;

import java.util.Random;

public class exercicio02 {
    public static void main(String[] args) {
        int[] vetor = new int[20];

        gerarValores(vetor, 0, 100);
        imprimirPares(vetor);
    }

    public static void gerarValores(int[] vetor, int min, int max) {
        Random rd = new Random();

        for (int i = 0; i < vetor.length; i++) {
            vetor[i] = rd.nextInt(max - min) + min;
        }
    }

    public static void imprimirPares(int[] vetor) {
        for (int i = 0; i < vetor.length; i++) {
            if (vetor[i] % 2 == 0) {
                System.out.println(vetor[i]);
            }
        }
    }
}
